docker-compose.yml

```diff
@@ -52,10 +53,11 @@ services:
     environment:
       - TPP_URL=http://open-banking-gateway:8085
 
-      - SPRING_DATASOURCE_URL=jdbc:h2:mem:test
-      - SPRING_DATASOURCE_USERNAME=SA
-      - SPRING_DATASOURCE_PASSWORD=
-      - SPRING_DATASOURCE_DRIVER-CLASS-NAME=org.h2.Driver
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://postgres:5432/open_banking?currentSchema=fintech
+      - SPRING_DATASOURCE_USERNAME=postgres
+      - SPRING_DATASOURCE_PASSWORD=docker
+      - SPRING_DATASOURCE_DRIVER-CLASS-NAME=org.postgresql.Driver
+      - SPRING_liquibase_enabled=true
```

Comment:
Doesn't seem to be needed as `application.yml` has: `enabled: true` for liquibase